### PR TITLE
[IMP] runbot-addons: add docker-image in runbot build

### PR DESCRIPTION
**Pull request by mapuerta**

### CODE REVIEW

File: runbot_travis2docker/models/runbot_build.py [hunk 1]
```diff
@@ -82,6 +82,7 @@ def _get_introspection(self):
     docker_executed_commands = fields.Boolean(
         help='True: Executed "docker exec CONTAINER_BUILD custom_commands"',
         readonly=True, copy=False)
+    docker_image_name = fields.Char("Docker image")
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py]: `default=vauxoo/odoo-80-image-shippable-auto`

**Comment by mapuerta** [comment]
@moylop260 Ready for review

Configured repository and docker image

config_repo

and created the builds, here show the log of test 

log_t2d

**Comment by moylop260** [comment]
Could you create a video where your custom image created from deployv works?

**Comment by mapuerta** [comment]
@moylop260 Here [video](https://www.youtube.com/watch?v=ZbkViCgNl64&feature=youtu.be) with configured deployv image in runbot

**Comment by moylop260** [comment]
What is the next step to use our deployv image in runbot?

**Comment by moylop260** [comment]
@JesusZapata Could you meeting with @mapuerta to see this matter together?

**Comment by JesusZapata** [comment]
@moylop260 @mapuerta 
Only one doubt

If the same repository has different image for each branch How to configure that custom image?

**Comment by mapuerta** [comment]
@JesusZapata It is good question. I think we missed this @moylop260  because each version must to have  custom  image. I think that we could add new field where indicate the version that work with that version  

@ruiztulio @moylop260 What do you think about this ?

**Pull request merged**